SharedPackages/BrowserServicesKit/Sources/NetworkProtection/Diagnostics/NetworkProtectionError.swift

```diff
@@ -60,7 +60,7 @@ public enum NetworkProtectionError: LocalizedError, CustomNSError {
     case setWireguardConfig(Error)
 
     // Auth errors
-    case noAuthTokenFound
+    case noAuthTokenFound(Error?)
```

Comment:
Adding underlying error information where available, as this is useful to have in pixels.